plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -65,6 +67,8 @@ func TestWriteWithDefaults(t *testing.T) {
 		},
 	}
 
+	require.NoError(t, logger.SetupLogging(&logger.Config{Debug: true}))
+	log.Logger = newLogger(i.Log).WithGroup("tcg.sdk")
```

Comment:
This should **not** be necessary when you call `i.Init()`! Please try to get rid of this and instead so what Telegraf would do. Deviate only for a really good reason! Same below.

Reply:
well, I've removed these parts and added test to cover logger integration instead